Create Calmcoder.xml

ctrlaltvikas opened the pull request:
A simple,elegant and colorful color-theme for coders.

ctrlaltvikas committed:
Create Calmcoder.xml

A simple,elegant and colorful color-theme for coders.

ricardograca commented:
Thanks! I'll check it out later today or tomorrow and either respond back with any questions/issues or merge it.

ricardograca commented:
After reviewing it (sorry it took this long) I'm not a big fan of the particular shade of purple used for some keywords. This makes them quite a bit hard to read against the green background:

![Calm Coder gedit theme preview](https://user-images.githubusercontent.com/1457035/34164083-83ec6662-e4d0-11e7-9658-b2b1b69f93df.png)

Everything else looks good to me. Do you have any intention of reviewing this?

vikaskyadav commented:
Yeah. I will be more than happy. Suggest me some changes.  I will try to
implement those within this week.!

On Tue, Dec 19, 2017 at 8:54 PM, Ricardo Graça <notifications@github.com>
wrote:

> After reviewing it (sorry it took this long) I'm not a big fan of the
> particular shade of purple used for some keywords. This makes them quite a
> bit hard to read against the green background:
>
> [image: Calm Coder gedit theme preview]
> <https://user-images.githubusercontent.com/1457035/34164083-83ec6662-e4d0-11e7-9658-b2b1b69f93df.png>
>
> Everything else looks good to me. Do you have any intention of reviewing
> this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ricardograca/gedit-themes/pull/13#issuecomment-352790786>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMBh16hHaqnw7s3coPISj_YEwnwVsi_4ks5tB9UbgaJpZM4IxNFV>
> .
>

ricardograca commented:
Just the purple color that's a bit hard to read. Everything else looks good.

ctrlaltvikas committed:
s

I think this might work. Although, I almost forgot this repo, since it has been so long time. :p 
I think new shade might do. If it doesn't work, you can replace purple with any suitable shade of your choice.

ricardograca commented on Calmcoder.xml:
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+  <?xml version="1.0" encoding="UTF-8"?>
```
These two spaces you added here made the XML document invalid and caused it to not be detected by Gedit.

ricardograca commented on Calmcoder.xml:
```diff
@@ -45,20 +45,20 @@
   <style name="def:doc-comment" foreground="#chocolate"/>
   <style name="def:doc-comment-element" foreground="#chocolate"/>
-  <style name="def:error" foreground="#f8f8f0" background="#f92672"/>
+  <style name="def:error" foreground="#f8f8f0" background="#ffcc66"/>
```
Looks good to me, good job :+1:

ricardograca reviewed (changes requested):
After you fix the invalid XML issue I'll merge it. Looks good to me otherwise.

ctrlaltvikas committed:
updated calmCoder file.

removed the irregular spacing and prettified it.
